godot-macros/src/class/godot_api.rs

```diff
@@ -230,10 +233,18 @@ fn process_godot_fns(decl: &mut Impl) -> Result<(Vec<FuncDefinition>, Vec<Functi
             }
 
             match attr.ty {
-                BoundAttrType::Func { rename } => {
+                BoundAttrType::Func { rename, gd_self } => {
                     // Signatures are the same thing without body
-                    let sig = util::reduce_to_signature(method);
-                    func_definitions.push(FuncDefinition { func: sig, rename });
+                    let mut sig = util::reduce_to_signature(method);
+                    if gd_self {
+                        sig.params = sig.params.clone();
```

Comment:
This statement seems pointless.

Reply:
Oh yeah, the clone is vestigial from a previous way I was doing this.